src/rust_inspector/binary.py

```diff
@@ -0,0 +1,278 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/rust-inspector for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import os
+import lief
+import json
+import zlib
+
+from typecode import contenttype
+from typecode.contenttype import get_type
+
+from rust_inspector.blint_binary import parse_symbols
+from rust_inspector.config import SPLIT_CHARACTERS_RUST
+from rust_inspector.config import STANDARD_SYMBOLS_RUST
+
+
+def is_macho(location):
```

Comment:
Eventually this function should end up be part of the typecode. Create a reminder issue

Reply:
This is there with https://github.com/aboutcode-org/typecode/issues/39